hack/lib/util.sh

```diff
@@ -18,6 +18,20 @@ kube::util::sortable_date() {
   date "+%Y%m%d-%H%M%S"
 }
 
+# arguments: target, item1, item2, item3, ...
+# returns 0 if target is in the given items, 1 otherwise.
+kube::util::array_contains() {
```

Comment:
My litmus test for "should I write this in shell" has always been "once you pass an array to a function, the answer becomes no".  I don't see an immediately better answer here, but this sets off all sorts of alarms.